### PR TITLE
Dynamic dashboard

### PR DESCRIPTION
**Pull request by AleBravoP**
Added DB table for categories pictures and made the Popular Items, as well as the Categories sections dynamic.

### CODE REVIEW

File: scripts/seed.js [hunk 1]
```diff
@@ -41,6 +42,59 @@ const seedCategories = async (client) => {
   return;
 }
 
+
+const seedCategoryPictures = async (client) => {
```
Comment (MATTALUI) [inline scripts/seed.js:46]: This works, but I don't think these need to be stored in the DB. They could probably just be images in the client.

File: src/app/dashboard/page.tsx [hunk 2]
```diff
@@ -1,83 +1,25 @@
...
-        <img
-            src="https://picsum.photos/1200/300"
-            alt="Hero Image"
-            className="w-auto 0 h-auto sm:mx-40 mx-auto object-cover"
-        />
+        <div
+            className="sm:mx-40 mx-auto w-auto bg-cover bg-bottom"
+            style={{
+                height: '300px',
+                backgroundImage: 'url(https://images.pexels.com/photos/3654776/pexels-photo-3654776.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=1)',
```
Comment (MATTALUI) [inline src/app/dashboard/page.tsx:13]: Why use a div with an image background over just using an image?

File: src/app/dashboard/page.tsx [hunk 3]
```diff
@@ -1,83 +1,25 @@
...
+            }}
+        ></div>
+        <Suspense>
```
Comment (MATTALUI) [inline src/app/dashboard/page.tsx:16]: We can do this in another PR, but I think these suspended components need a loading skeleton fallback in order to look nice.

**Review by MATTALUI — APPROVED** [review]

File: src/app/dashboard/page.tsx (hunk 2 above)
Comment (AleBravoP) [reply src/app/dashboard/page.tsx:13]: This is to keep a banner size, the original picture is larger than 300px
